scipy/stats/tests/test_stats.py

```diff
@@ -6042,8 +6050,8 @@ def test_normalitytests():
     # see gh-13549.
     # skew parameter is 1 > 0
     a1 = stats.skewnorm.rvs(a=1, size=10000, random_state=123)
-    pval = stats.skewtest(a1, alternative='greater').pvalue
-    assert_almost_equal(pval, 0.0, decimal=5)
+    pval = stats.skewtest(xp.asarray(a1), alternative='greater').pvalue
+    xp_assert_close(pval, xp.asarray(0.0, dtype=xp.float64), atol=1e-7)
```

Comment:
A slow morning but I could work out how to get this to play nicely with the 0 desired value without
```
E   Relative difference: inf (up to 1e-07 allowed)
```

Reply:
Yep, we need to add an `atol` in when we desire `0`.

Reply:
Ha yes clearly I was not properly awake this morning